zcash_client_backend/src/keys.rs

```diff
@@ -107,10 +109,11 @@ impl UnifiedSpendingKey {
 
     pub fn to_unified_full_viewing_key(&self) -> UnifiedFullViewingKey {
         UnifiedFullViewingKey {
-            account: self.account,
             #[cfg(feature = "transparent-inputs")]
             transparent: Some(self.transparent.to_account_pubkey()),
```

Comment:
If the `transparent-inputs` feature is not enabled, will any transparent component be dropped (which seems incorrect), or will it be stored in `unknown` (to preserve round-trip serialisation)?